src/pages/projects/[id].tsx

```diff
@@ -162,7 +160,34 @@ function Project() {
     const result = await chainService.submitMilestone(
       account,
       onChainProject,
-      mileStoneKeyInView
+      milestoneKeyInView
+    );
+    while (true) {
```

Comment:
Do we really need this while loop ?

Reply:
we need a way to ensure loading is false while we wait for the chain to return a status
Is there a better way of doing that?